neurodamus/core/random.py

```diff
@@ -3,57 +3,36 @@
 """
 from __future__ import absolute_import
 from . import Neuron
-from neurodamus.core.configuration import RNGConfig
 
 
-# NOTE: These are pure wrappers, in the sense they dont create Python objects. Instead
+# NOTE: These are pure wrappers, in the sense they don't create Python objects. Instead
 #       Neuron objects are returned (using __new__)
 
 
 class RNG(object):
-    def __new__(cls, *ids, **kw):
+    def __new__(cls, **kw):
         """Creates a default RNG, currently based on ACG"""
```

Comment:
Just discussed with @alkino  and @pramodk , it looks good that we instantiate Random123 object with a (0,1,2, seed), the first 3 seeds will be changed later on during synapse creation by gid, syn_id, etc.

Another nitpick, could you please change the docstring to replace ACG with Random123 ?